test-lib.sh

```diff
@@ -788,4 +788,53 @@ ct_check_latest_imagestreams() {
     python3 "${test_lib_dir}/check_imagestreams.py" "$latest_version"
 }
 
+# ct_test_app_dockerfile
+# -----------------------------
+# Argument: dockerfile - path to a Dockerfile that will be used for building an image
+#                        (must work with an application directory called 'app-src')
+# Argument: app_url - git URI with a testing application
+# Argument: body_regexp - PCRE regular expression that must match the response body
+# Argument: port - Optional port number (default: 8080)
+ct_test_app_dockerfile() {
+  local dockerfile=$1
+  local app_url=$2
+  local expected_text=$3
+  local port=${4:-8080}
+  local app_dir=app-src # this is a directory that must match with the name in the Dockerfile
```

Comment:
```suggestion
  local app_dir="app-src" # this is a directory that must match with the name in the Dockerfile
```

Reply:
I've changed this -- it's now an argument that must be configured.